archesproject/arches
# 7785 cbyrd cache user permissions

### Types of changes


### Description of Change

Caches user permissions

### Issues Solved

#7785 


### To Test

1. Create a user with all privileges, but is not a superuser.
2. Ensure you are running the `silk` DB profiling plugin
3. Compare page load times and DB queries across the application between this branch and master
4. Ensure permissions filtering still works via search
5. Ensure cache busting works via:

   - interaction with the modal in admin templates
   - adding a node to a graph
   - changing node permissions via the permissions manager in the Graph Designer

       Note: There should be a difference in the number of DB queries in a page load with a busted cache vs a non-busted cache.


### Checklist
